decentraland/unity-renderer
# fix: channel member labels, invalid characters in notification panel & channels not loading

## What does this PR change?

- Changed channel entry labels to "members joined" depending on a feature flag.
- Chat notification panel replaces invalid characters for `?`
- Fixed that sometimes channels were not being listed in the conversation list

## How to test the changes?
1. Open the conversation list as fast as possible, channels should be listed correctly. You may have a delay due servers latency.
2. Get someone to write invalid characters in chat like: rrrŕŗrŗrrrrŕrṙrrr̶̽̅rrrŕŗrŗrrrrŕrṙrrŗ
3. When you receive the notification in the panel, characters should be replaced by `?`
4. Add `&ENABLE_matrix_presence_disabled` to the test url. The channels in the conversation list should say "members joined" instead of "members online"

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
